HISTORY.md

```diff
@@ -1,5 +1,10 @@
 # Release History - open AEA
 
+## 1.59.0 (2024-10-29)
+
+Plugins:
+- Fixes the gas estimation #766
+
```

Comment:
should we include #764 also?

Reply:
3fc28c1292807bc2caf33b4c726dead6bded88b9